MANIFEST.in

```diff
@@ -1 +1 @@
-recursive-include flamby/*.csv
\ No newline at end of file
+recursive-include flamby/ *.csv
```

Comment:
```suggestion
recursive-include flamby *.csv
```

Reply:
Sounds like it would be even more idiomatic to write it like that, what do you think ?

Reply:
yes makes total sense, thank you! I had been lazy to just introduce the space, but your version is much cleaner :)